src/config.js

```diff
@@ -76,22 +76,30 @@ const config = convict({
     }
   },
   httpProxy: {
-    doc: 'HTTP Proxy URL',
-    format: String,
-    nullable: true,
-    default: null,
-    env: 'HTTP_PROXY'
+    isEnabled: {
+      doc: 'Is http proxy enabled',
+      format: Boolean,
+      default: false,
+      env: 'HTTP_PROXY_ENABLED'
```

Comment:
It would be more idiomatic with how `HTTP_PROXY` works if its just based on weather or not the environment variable is set. `HTTP_PROXY` is a long standing and widely understood convention where as `HTTP_PROXY_ENABLED` is cdp specific.
Principle of least surprises etc.